packages/@expo/cli/src/start/platforms/ios/AppleDeviceManager.ts

```diff
@@ -141,8 +143,9 @@ export class AppleDeviceManager extends DeviceManager<SimControl.Device> {
   }
 
   private async getApplicationIdFromBundle(filePath: string): Promise<string> {
-    // TODO: Implement...
-    return EXPO_GO_BUNDLE_IDENTIFIER;
+    const builtInfoPlistPath = path.join(filePath, 'Info.plist');
+    const { CFBundleIdentifier } = await parsePlistAsync(builtInfoPlistPath);
```

Comment:
Can you do a fs.exists check and `Log.warn` if the file cannot be found, then fallback on the Expo Go bundle id, I can do the rest (testing, debug logging) after merged if you'd like. Otherwise, you can add with a line like:
```js
const debug = require('debug')('expo:start:platforms:ios:AppleDeviceManager') as typeof console.log;
```
at the top and a `debug('getApplicationIdFromBundle:', filePath);` makes it a bit easier for users to find out what's going on.